api/database/factories/ModelFactory.php

```diff
@@ -13,25 +13,6 @@
 
```

Comment:
Unrelated to this pull request, but are we able to segregate this file to stop it from becoming massive and procedural?